resources/js/Pages/Admin/Rooms/CreateRoomForm.vue

```diff
@@ -76,6 +76,28 @@
                         <jet-input-error :message="createRoomForm.error('status')" class="mt-2"/>
                     </div>
 
+                
+                    <div class="col-span-6 sm:col-span-3">
+                              <jet-label for="room_type" value="Room type" />
+                              <select
+                                v-model="createRoomForm.room_type"
+                                class="mt-1 block w-full"
+                                name="room_type"
+                                id="room_type"
+                              >
+                                <option value="" selected disabled hidden>
+                                  Select Room Type
+                                </option>
+                                <option value="lounge">Lounge</option>
+                                <option value="mezzanine">Mezzanine</option>
+                                <option value="conference">Conference</option>
+                              </select>
```

Comment:
The list of room types gets passed but it is hard coded here?

Reply:
ur right it shouldnt be will address this